drivers/include/slipdev.h

```diff
@@ -120,13 +129,19 @@ typedef struct {
 typedef struct {
     netdev_t netdev;                        /**< parent class */
     slipdev_params_t config;                /**< configuration parameters */
-    chunk_ringbuf_t rb;                     /**< Ringbuffer to store received frames.       */
+    chunk_ringbuf_t rb;                     /**< Ringbuffer to store received NET frames.   */
```

Comment:
Is “NET frames” wording from the draft?

Reply:
no it is not, it would be "Packet Transfer". I thought I would assume a bit that the reader is more likely to know RIOT than the Draft, so I picked a name that represents "Packet Transfer" for RIOT networking humans. Do you think that was a poor choice? I have no hard feelings on this. ("Packet frames" is rather confusing as one could say "coap packet")

Reply:
ah yes, I can see that as confusing. I applied your suggestion.